lib/auth/auth.go

```diff
@@ -35,6 +35,7 @@ import (
 	"time"
 
 	"github.com/gravitational/teleport"
+	enterpriseevents "github.com/gravitational/teleport/e/lib/events"
```

Comment:
I don't think we can add direct dependencies on `teleport/e/...` in the main codebase.
Otherwise an OSS version can't compile without the local `e` repo checked out.

Either put these events in the main repo, or move all of roles' CRUD logic to `e`.

Reply:
Yeah, we 100% need to be able to compile OSS without `e` at all times.

Reply:
revised: moved events into main repo